build

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
This file is not related to any of the changes we are doing, we can safely delete this file.